middleware/http/transport.go

```diff
@@ -57,6 +73,13 @@ func TransportTrace(enable bool) TransportOption {
 	}
 }
 
+// TransportTrace allows one to enable Go's net/http/httptrace.
```

Comment:
TransportErrHandler ...

Reply:
Interesting, maybe we need a lint for godoc?